automated_analysis.py

```diff
@@ -50,11 +50,10 @@
 
     messages_json_input_path = args.messages_json_input_path
     individuals_json_input_path = args.individuals_json_input_path
-    output_dir = args.output_dir
+    automated_analysis_output_dir = args.automated_analysis_output_dir
 
-    IOUtils.ensure_dirs_exist(output_dir)
-    IOUtils.ensure_dirs_exist(f"{output_dir}/maps")
-    IOUtils.ensure_dirs_exist(f"{output_dir}/graphs")
+    IOUtils.ensure_dirs_exist(automated_analysis_output_dir)
+    IOUtils.ensure_dirs_exist(f"{automated_analysis_output_dir}/graphs")
 
     log.info("Loading Pipeline Configuration File...")
     with open(pipeline_configuration_file_path) as f:
```

Comment:
L65: You no longer need to download the drive service credentials here

Reply:
thanks